scripts/register-completions.ps1

```diff
@@ -13,3 +13,16 @@ Register-ArgumentCompleter -Native -CommandName xmake -ScriptBlock {
     }
     $env:XMAKE_SKIP_HISTORY = $oldenv
 }
+Register-ArgumentCompleter -Native -CommandName xrepo -ScriptBlock {
+    param($commandName, $wordToComplete, $cursorPosition)
+    $complete = "$wordToComplete"
+    if (-not $commandName) {
+        $complete = $complete + " "
+    }
+    $oldenv = $env:XMAKE_SKIP_HISTORY
+    $env:XMAKE_SKIP_HISTORY = 1
+    xmake lua --root private.xrepo.complete "0" "nospace" "$complete" | ForEach-Object {
+        [System.Management.Automation.CompletionResult]::new($_, $_, 'ParameterValue', $_)
+    }
```

Comment:
xmake now can export json completion data (which include name and description) which powershell can convert and exploit
[see here](https://github.com/xmake-io/xmake/pull/3214#issuecomment-1366572352) for examples